.github/workflows/generate-embeddings.yml

```diff
@@ -0,0 +1,17 @@
+name: "Generate Embeddings"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate-embeddings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: eic/trigger-gitlab-ci@v3
+        with:
+          url: https://git.tools.mia-platform.eu
+          project_id: ${{ vars.MIA_ASSISTANT_GITLAB_PROJECT_ID }}
+          token: ${{ secrets.MIA_ASSISTANT_GITLAB_TOKEN }}
```

Comment:
This token should be created before merging the PR @davidebianchi  FYI

Reply:
actually we can merge and only wait for the trigger but let's make sure the secret name is ok

Reply:
Added to the repo

Reply:
Tokens have been created.